app/assets/javascripts/components/admin/roster_page.es6.jsx

```diff
@@ -22,10 +24,29 @@ class RosterPage extends React.Component {
     );
   }
 
+  onPersonClick(person) {
+    this.setState({ showPersonModal: true });
```

Comment:
you can just put this into one, so `this.setState({ showPersonModal: true, person: person });